### PR TITLE
prevent highlighting of menu items

### PR DESCRIPTION
**Pull request by Kevingislason**
## Description

closes: #568

Also fixes a style issue for the zkAccount button as displayed when the signer is out of date I found while working on this PR

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [x] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [x] Updated relevant documentation in the code
- [x] Test code changes manually, and describe your procedure in a comment on this PR
- [x] Mark this PR with the correct milestone

### CODE REVIEW

**Review by jasonintju — COMMENTED** [review]
The line5 can be deleted at the same time.
`import { useTxStatus } from 'contexts/txStatusContext';`

File: src/components/Accounts/AccountSelectMenu.tsx [hunk 1]
```diff
@@ -32,14 +30,14 @@ const DisplayAccountsButton = () => {
     return (
       <>
         <img
-          className="w-6 h-6 rounded-full"
+          className="unselectable-text w-6 h-6 rounded-full"
```
Comment (jasonintju) [inline src/components/Accounts/AccountSelectMenu.tsx:33]: In fact, there is a classname called 'select-none' in Tailwindcss v2.

**Review by jasonintju — APPROVED** [review]

File: src/components/Accounts/AccountSelectMenu.tsx (hunk 1 above)
Comment (jincan39) [reply src/components/Accounts/AccountSelectMenu.tsx:33]: agree with Jason, converting all 'unselectable-text' to 'select-none' would be more tailwindify.

**Review by jincan39 — APPROVED** [review]

File: src/components/Accounts/AccountSelectMenu.tsx (hunk 1 above)
Comment (Kevingislason) [reply src/components/Accounts/AccountSelectMenu.tsx:33]: `unselectable-text` applies some additional styles  that `select-none` doesn't, which I think handle certain less common browsers

**Pull request merged**